### DISCUSSION TITLE
Initiative not required still auto-sort

### DISCUSSION THREAD

**Discussion by Bnibling** [Lurkars/gloomhavensecretariat#154] [Ideas]
It would be nice if "Initiative is required" is unchecked and the draw would still auto sort the minions by initiative with character remaining on top.

We still manage our characters initiative manually, but would like to use the app to handle all the minions and loot for our characters. At the moment, we play without the characters on the app and manually handle loot and character stuff, which is fine, but this would be a nice little easy of use addition.

**Comment by Lurkars** [comment]
Yes, I think old helper behaved same. I will change that!

**Comment by Lurkars** [comment]
Should be implemented now in [v0.49.3](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.49.3). I also added a setting to disable sorting in general, don't know if some are using the app different, so leave that option.

**Reply by Bnibling** [reply]
Just tested and works great! Thanks!
